src/programs/console_test/console_test.cpp

```diff
@@ -497,6 +501,51 @@ void MyTestApp::TestElectronExposureFilter()
 	EndTest();
 }
 
+void MyTestApp::TestEmpiricalDistribution()
+{
+  BeginTest("Test Empirical Distribution");
```

Comment:
This is not crucial at all, but `BeginTest("Empirical Distribution");` might be nicer so the output of console_test is not 
` Testing Test Empirical Distribution                   : PASSED!`

Reply:
Good point! Since it is minor, I'll change that next go round